p2p/net/mock/mock_stream.go

```diff
@@ -121,6 +121,15 @@ func (s *stream) Close() error {
 	return nil
 }
 
+func (s *stream) CloseRead() error {
+	return s.read.CloseWithError(ErrClosed)
+}
+
+func (s *stream) Close() error {
+	s.CloseRead()
```

Comment:
do we need `_ = ` to satisfy the linter?

Reply:
We don't have that linter here, but it can't hurt.